common/static/common/templates/components/paging-header.underscore

```diff
@@ -1,8 +1,23 @@
 <% if (!_.isUndefined(srInfo)) { %>
   <h2 class="sr" id="<%= srInfo.id %>"><%- srInfo.text %></h2>
 <% } %>
-<div class="search-tools">
-    <span class="search-count">
+<div class="search-tools listing-tools">
+    <span class="search-count listing-count">
         <%= message %>
     </span>
+    <% if (showSortControls) { %>
```

Comment:
It would be good to have a few Jasmine tests to verify that this component renders correctly. Have a test with not showing controls, and then a test showing controls and with different selected values.